src/WebApi/Controllers/UsersController.cs

```diff
@@ -448,7 +448,7 @@ public Task<ActionResult<Result<UserItemViewModel>>> BuyItem([FromBody] BuyItemC
     /// Gets user clan or null.
     /// </summary>
     [HttpGet("self/clans")]
-    public Task<ActionResult<Result<ClanViewModel>>> GetUserClan()
+    public Task<ActionResult<Result<UserClanViewModel>>> GetUserClan()
```

Comment:
It looks like it's the relevant change to the PR here. Could you please discard the rest.

Reply:
Whole ClanViewModel and references u mean?

Reply:
ah i overlooked the autoformatting stuff, sorry, added 6ca523d to revert layout changes